src/backend/routes/bookingRoutes.ts

```diff
@@ -100,6 +100,17 @@ router.get<Record<string, never>, { data: Array<ApiBooking> } | { error: string
   },
 );
 
+router.get<Record<string, never>, { data: Array<ApiBooking> } | { error: string }>(
+  '/user/:userUuid',
+  isAuthenticated([UserRole.staff, UserRole.volunteer]),
```

Comment:
I think volunteers should not have access to bookings of other users

```suggestion
  isAuthenticated([UserRole.staff]),
```